### PR TITLE
Developer layer example

### PR DESCRIPTION
**Pull request by AdamIsrael**
Fix some papercuts and edit for clarity

### CODE REVIEW

File: src/en/developer-layer-example.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ## Writing a layer by example
 
-In this howto, we will be writing a charm layer for Vanilla. Vanilla is an open
-source themeable, pluggable, multi-lingual forum software, which enables
-communities to host a forum for discussion topics at scale. Powered by PHP and
-MySQL Vanilla is a fine example of a 3 tiered application
+In this document we will be writing a charm layer for Vanilla. Vanilla is an
+open source, themeable, pluggable, and multi-lingual forum software, which
```
Comment (chuckbutler) [inline src/en/developer-layer-example.md]: This feels like a runaway run-on now

**Comment by chuckbutler** [comment]
Minor comment left inline, otherwise :+1: LGTM

File: src/en/developer-layer-example.md (hunk 1 above)
Comment (AdamIsrael) [inline src/en/developer-layer-example.md]: First sentence and the dropped comma?

Comment (chuckbutler) [inline src/en/developer-layer-example.md]: Yeah, its 2 logical thoughts mashed together with verbs and nouns and all that goodness. Maybe i'm being too picky, when read aloud it sounds fine.

Comment (AdamIsrael) [inline src/en/developer-layer-example.md]: I don't think it's technically wrong but I lean towards more commas in general.

**Comment by evilnick** [comment]
:frog:

**Pull request merged**